.github/workflows/windows.yml

```diff
@@ -111,16 +34,16 @@ jobs:
           PATH: '${env:PATH};C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64'
```

Comment:
Interesting Microsoft is still installing Visual Studio into the 32-bit program files area (I would have thought it would be similar to GCC.. a pure 64-bit program, but can target both 64-bit and 32-bit binaries).